CHANGELOG.md

```diff
@@ -11,9 +11,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - modern issue template structure
 - `--info` flag in CLI
 ### Changed
+- `bug_report.yml`
```

Comment:
Please eliminate these two lines.
These changes are not related to the package itself.